src/plugins/field_formats/server/ui_settings.ts

```diff
@@ -71,6 +72,12 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
           params: schema.object({}),
           es: schema.boolean(),
         }),
+        geo_point: schema.object({
```

Comment:
I am not sure how we normally handle this, but because of this users who have non-default `defaultTypeMap` will have it not valid anymore and they will have a warning in logs: 

```
[2021-12-16T15:18:06.993+01:00][WARN ][ui-settings-service] Ignore invalid UiSettings value. Error: [validation [format:defaultTypeMap].geo_point.id]: expected value of type [string] but got [undefined].
```


One way it to make this field optional, but then those non-default users will miss out on this new feature. 
So I guess we should add a migration instead: 

https://github.com/elastic/kibana/blob/main/src/core/server/ui_settings/saved_objects/migrations.ts

This migration would add `"geo_point": { "id": "geo_point", "params": { "transform": "wkt" } }` to `defaultTypeMap`overrides